exasol/ds/sandbox/runtime/ansible/roles/jupyter/files/notebook/script_languages_container/using_a_script_languages_container.ipynb

```diff
@@ -0,0 +1,704 @@
+{
```

Comment:
<p>"The remaining build steps should be only changed with care, but sometimes some dependencies are defined in other build steps because the script client depends on them. "</p><p>This is a long and a bit awkward sentence. Changing the other build steps is required only in rare cases. Caution is needed when doing so. A typical use case is when dependencies are defined in other build steps because the script client depends on these dependencies.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/exasol/ai-lab/pull/300/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='21'/>

Reply:
<p>Used your suggestion</p>